molecule/default/tests/test_default.py

```diff
@@ -14,7 +14,11 @@
 
 @pytest.mark.parametrize(
     "f",
-    ["/root/.cobaltstrike.license", "/opt/cobaltstrike", "/opt/cobaltstrike/update"],
+    [
+        "/opt/cobaltstrike/cobaltstrike.auth",
+        "/opt/cobaltstrike",
+        "/opt/cobaltstrike/update",
```

Comment:
Just updating the sorting for these with the value change.

```suggestion
        "/opt/cobaltstrike",
        "/opt/cobaltstrike/cobaltstrike.auth",
        "/opt/cobaltstrike/update",
```